kin-core/src/main/java/kin/core/KinClient.java

```diff
@@ -117,6 +120,7 @@ public boolean hasAccount() {
     /**
      * Returns the number of existing accounts
      */
+    @SuppressWarnings("WeakerAccess")
```

Comment:
why did you add this now ?

Reply:
those are old false warnings that I got tired of (those methods are public intentionally though are not used in sample app)